.github/workflows/include_supported-versions.yml

```diff
@@ -0,0 +1,50 @@
+name: Load Supported Versions
```

Comment:
Maybe I didn't understand correctly, but aren't `include_latest-version` and `include_supported-versions.yml` a bit of a duplication ?

The only difference I can sport is `sort | reverse` VS `sort | .[-1]` for both TF and AWSCLI versions, I'm not sure to quite understand the use case.

EDIT : 
Oh okay the difference is the latest version for each VS a matrix of all versions for each tool. 

In that cas, a plural on outputs would be nice 😉